`capdl_linker.py` is not executable

capDL is introduced in the threads tutorial but it appears that the logic for running the `capdl_linker.py` script via cmake may be broken. AFAICT a clean checkout of the tutorials and capdl repositories, and a clean `../init --tut threads` results in a build error when we try to execute the script with its `#!` rather than explicitly via `python`.

```
[1/12] cd /path/to/sel4-tutorials/threads_build && /usr/bin/cmake -E env PYTHONPATH=/path/to/sel4-tutorials/projects/capdl/python-capdl-tool /path/to/sel4-tutorials/projects/capdl/cdl_utils/capdl_linker.py --arch=x86_64 --object-sizes /path/to/sel4-tutorials/threads_build/capdl/object_sizes/object_sizes.yaml build_cnode --manifest-in=/path/to/sel4-tutorials/threads/.manifest.obj --elffile threads --ccspace /path/to/sel4-tutorials/threads_build/cspace_threads.c
FAILED: cspace_threads.c
cd /path/to/sel4-tutorials/threads_build && /usr/bin/cmake -E env PYTHONPATH=/path/to/sel4-tutorials/projects/capdl/python-capdl-tool /path/to/sel4-tutorials/projects/capdl/cdl_utils/capdl_linker.py --arch=x86_64 --object-sizes /path/to/sel4-tutorials/threads_build/capdl/object_sizes/object_sizes.yaml build_cnode --manifest-in=/path/to/sel4-tutorials/threads/.manifest.obj --elffile threads --ccspace /path/to/sel4-tutorials/threads_build/cspace_threads.c
Permission denied
ninja: build stopped: subcommand failed.
```

`chmod u+x .../capdl_linker.py` fixes the build. It's unclear to me if this is the the tutorial's cmake logic's fault or if the capdl repo should be changed to make the script executable.

--- a/cmake/helpers.cmake
+++ b/cmake/helpers.cmake
@@ -165,6 +165,7 @@ endfunction()
 
 find_package(capdl REQUIRED)
 file(GLOB_RECURSE capdl_python ${PYTHON_CAPDL_PATH}/*.py)
+set(PYTHON3 "python3" CACHE INTERNAL "")
 
 set(python_with_capdl ${CMAKE_COMMAND} -E env PYTHONPATH=${PYTHON_CAPDL_PATH} ${PYTHON3})
 set(capdl_linker_tool ${python_with_capdl} ${CAPDL_LINKER_TOOL})

